static/allowlist.json

```diff
@@ -16,7 +16,7 @@
     "https://gateway.ipfs.io",
     "wss://*.g.alchemy.com",
     "https://*.infura.io",
-    "https://burned-solitary-breeze.bsc.quiknode.pro",
+    "https://thrumming-shy-lake.bsc-testnet.quiknode.pro",
```

Comment:
It seems to me you should be adding a new line instead of replacing prod with testnet